Symfony/CS/Tests/Fixtures/Integration/priority/phpdoc_no_package,phpdoc_regular_header.test

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Integration of fixers: phpdoc_regular_header,header_comment.
+--CONFIG--
+level=none
+fixers=phpdoc_no_package,phpdoc_regular_header
+--EXPECT--
+<?php
+/*
+ * Docblock without phpdoc
+ *
```

Comment:
But that completely defeats the purpose? Why would we leave docblock annotations in a non-docblock? In that order we would get from this case:
```php
<?php
/*
 * @package thisNeverGetsReadOrParsedByAnyonee
 */
```